spyder/utils/introspection/plugin_client.py

```diff
@@ -230,11 +233,11 @@ def handle_return(value):
         else:
             plugin.request('foo')
 
-    def handle_errored():
+    def handle_errored(name):
```

Comment:
The signal that connects with this function pass as an argument the name of the plugin. Should I use the name in the print statement?